book/src/plonk/commitments.md

```diff
@@ -1,60 +0,0 @@
-# Commitments
-
-## What's a commitment
-
-A commitment is a cryptographic primitive that allows you to lock "something" (perhaps a number) without revealing it (yet). This will produce a **commitment**. Later, someone can ask you to reveal the "something". After you do, then can verify that it is linked to the commitment by **opening** it.
-
-<img src="../../img/commitment.png" width="300px">
```

Comment:
I thought that was a helpful diagram o.o

Reply:
They don't load for me.  I can put it back in or, if you like, I can convert if there's a new way in latex/md?

Reply:
<img width="505" alt="commitment" src="https://user-images.githubusercontent.com/1316043/161615984-ce6fdddb-2dd4-4fcc-90ca-4a4ea67baaac.png">

ah, I might have messed up some images when I moved the book to this repo, it was this diagram

Reply:
Cool - I'll put it back in the simple section.  Does it look the way you want now (the one above)?

Reply:
I've made a latex version that support scaling and fixes the issue with the above diagram ("open" box should be "verify")